src/js/node/zlib.ts

```diff
@@ -141,6 +141,14 @@ ObjectDefineProperty(Zlib.prototype, "_strategy", {
 Zlib.prototype.params = function (level, strategy, callback) {
   return this[kHandle].params(level, strategy, callback);
 };
+Zlib.prototype._transform = function _transform(chunk, encoding, callback) {
+  try {
+    this[kHandle].transformWith(chunk, encoding, this, false);
+    callback();
```

Comment:
is the first argument supposed to be null in this case or is it supposed to be undefined?

Reply:
no https://nodejs.org/docs/latest/api/stream.html#transform_transformchunk-encoding-callback